source/extensions/quic_listeners/quiche/platform/quiche_test_impl.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
This file is already in test/extensions/..., which is correct since it's test only.
Can you remove this file?